gridstatus/tests/test_ercot2.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from gridstatus import Ercot, NotSupported
+from gridstatus.tests.base_test_iso import BaseTestISO
+
+
+class TestErcot(BaseTestISO):
+    iso = Ercot()
+
+    """get_fuel_mix"""
+
+    @pytest.mark.skip
```

Comment:
This is an extraneous decorator, but I thought it was a good way of noting the ISO is opting-out of a test in addition to the `pass`.

The other options I see are adding a decorator to the BaseTestISO method which includes or excludes certain subclasses, but at the point it seems like six of one, half of a dozen of the other.

Reply:
I like this approach. i think it's most scalable to have the subclass handle their own opting out. seems better if the base test class doesn't have to be aware of all it's subclasses 

one thought I have to make it even more clear is add a reason to the marker like this `@pytest.mark.skip(reason="Not Applicable")`.